src/comment.js

```diff
@@ -1,34 +1,26 @@
 import { commentFor } from './strategy.js'
 
-const getCommenter =
-  (filepath, options, logger = console) =>
-  (rgxMatch, capturedImportPath) => {
-    const importPath = capturedImportPath.trim()
-    const bareImportPath = importPath.replace(/['"`]/g, '')
-    const { verbose, match, ...magicCommentOptions } = options
-    const magicComment = Object.keys(magicCommentOptions)
-      .map(key => {
-        const option = magicCommentOptions[key]
-
-        if (option) {
-          return commentFor[key](filepath, bareImportPath, option, match)
-        }
```

Comment:
The use of `schema-utils` and validating the options negates the need for this check, thereby allowing the code to be simplified.